doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_nordic_bundles.rst

```diff
@@ -0,0 +1,38 @@
+.. _ug_nrf54h20_suit_nordic_bundles:
+
+SUIT Nordic Semiconductor Bundles
```

Comment:
```suggestion
nRF54h20 SOC binaries
```

Reply:
I do not really like this title, this article focuses more on the update process, it doesn't describe in depth how the bundles work.. I've changed my original title to "Upgrade of Nordic Semiconductor firmware components with SUIT" - is that OK?